testing/test-utils/src/lib/utils/test-folder-setup.unit.test.ts

```diff
@@ -0,0 +1,155 @@
+import { bold } from 'ansis';
+import { vol } from 'memfs';
+import * as fsPromises from 'node:fs/promises';
+import { describe, expect, it, vi } from 'vitest';
+import { MEMFS_VOLUME } from '@code-pushup/test-utils';
+import {
+  cleanTestFolder,
+  restoreNxIgnoredFiles,
+  teardownTestFolder,
+} from './test-folder-setup.js';
+
+describe('restoreNxIgnoredFiles', () => {
+  it('should rename Nx ignored files in a folder', async () => {
+    vol.fromJSON(
+      {
+        '/_nx.json': '{}',
+        '/_package.json': '{}',
+        '/_project.json': '{}',
+      },
+      MEMFS_VOLUME,
+    );
+
+    await expect(restoreNxIgnoredFiles('/')).resolves.not.toThrow();
+
+    expect(vol.existsSync('/nx.json')).toBe(true);
+    expect(vol.existsSync('/_nx.json')).toBe(false);
+    expect(vol.existsSync('/package.json')).toBe(true);
+    expect(vol.existsSync('/_package.json')).toBe(false);
+    expect(vol.existsSync('/project.json')).toBe(true);
+    expect(vol.existsSync('/_project.json')).toBe(false);
```

Comment:
How about using `vol.toJSON()` to list everything?

```suggestion
    expect(vol.toJSON()).toEqual({
      '/nx.json': '{}',
      '/package.json': '{}',
      '/project.json': '{}',
    });
```